kubernetes-cluster-eks/config.yml

```diff
@@ -0,0 +1,15 @@
+version: "3"
+containers:
+- name: workstation
```

Comment:
What is the reason to separate the workstation and the cloud-client container? You can also use the cloud-client as a workstation, or is there some other consideration here?